Kubernetes/HOWTO-on-prem.md

```diff
@@ -0,0 +1,103 @@
+# Kubernetes On-Premises Deployment From Scratch -- Start to Finish #
+This guide will walk you through deploying *Kubernetes 1.8* on a Linux master and join two Windows nodes to it without a cloud provider.
+
+**Note**: The guide will assume a local working directory of `~/kube` for the Kubernetes setup. If you choose a different directory, just replace any references to that path.
+
+## Building the Binaries ##
+We will begin with a Linux master node and a Windows worker node on RS3+, both of which are assumed to be VMs.

```

Comment:
RS3+ is the Windows node requirement, but what are the Linux master node requirements? Are we making any assumption on which distro of Linux the customer should be using to follow along with this guide?

Reply:
Good point -- I'll make note of it to just say "a recent Ubuntu-like distro".